Added listing for Bruce Harrell, mayor of Seattle

aubreyp1977 opened the pull request:
This is my first commit to the people project. I tried my best to follow the formatting schema, and submit the correct information for the person.

alexobaseki reviewed (dismissed):
Thank you for your contribution.

aubreyp1977 commented:
@alexobaseki Thank you for catching that! On disk my file is in the right place, so I must have made a mistake when committing it.

aubreyp1977 commented:
I moved the file to the correct location. I verified that the previous mayor was not listed in "municipalities" dir.

alexobaseki reviewed (approved):
Thank you!

alexobaseki commented:
> Thank you!

Your changes failed linting requirement. Here is the error
![image](https://github.com/user-attachments/assets/892790c0-9798-4f3c-b5eb-7aa4738c6790)
* Fixing "Wrong newline" I think the character that created the newline between line 1 and line 2 is wrong. Delete the new line and use the "return" or "enter" key to create it again.
* Delete any space behind any end of line. This is lines 11 and 15

aubreyp1977 commented:
I fixed the new line encoding and removed the trailing periods. However, I moved the change to a new branch, because this is the more appropriate workflow. I opened a new pull request from my new branch. Closing this pull request. Thank you!
